test/e2e/scoped_client_test.go

```diff
@@ -61,7 +61,7 @@ var _ = Describe("Scoped Client", func() {
 			// lack of permission.
 			name: "ServiceAccountDoesNotHaveAnyPermission",
```

Comment:
The purpose of the `name` field on `testParameter` is served by the first argument to each call to `table.Entry`, so it can be removed. And a nit: those entry descriptions could better describe the behavior under test. They're analogous to `It` descriptions -- Ginkgo generates an `It` for each table entry.